hack/testing-olm-upgrade/resources/cr.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "Elasticsearch"
+metadata:
+  name: "elasticsearch"
+  annotations:
+      elasticsearch.openshift.io/loglevel: trace
+spec:
+  managementState: "Managed"
+  nodeSpec:
+    resources:
+      limits:
+        memory: 1Gi
+      requests:
+        cpu: 100m
+        memory: 1Gi
+  nodes:
+  - nodeCount: 3
+    roles:
+    - client
+    - data
+    - master
+    storage:
+      storageClassName: gp2
+      size: 100G
```

Comment:
we can likely get away with `10G` for ci testing -- its what i use in my local cluster when testing

Reply:
Fixed